db/1.10/modules/ROOT/pages/database-connector-troubleshooting.adoc

```diff
@@ -163,6 +163,13 @@ To resolve this issue:
 
 Set the following system property to the deployment `-DdoTestConnectivity=false`.
 
+== Troubleshoot Oracle NULL values
+
+You may encounter problems when trying to send a `NULL` value to an Oracle database.
+
+To resolve this issue:
+
+In Oracle, an empty string equals to `NULL`, instead send `''`.
```

Comment:
Perhaps consider rewording to this: In Oracle, if an empty strings equals `NULL`, send `''` instead.